PowerEditor/src/NppDarkMode.cpp

```diff
@@ -2637,6 +2637,19 @@ namespace NppDarkMode
 		return false;
 	}
 
+	static void setSysLinkCtrlCtlColor(HWND hWnd, NppDarkModeParams p)
```

Comment:
Please change function name from "setSysLinkCtrlCtlColor" to "setUrlLinkControlColor"

Reply:
Agree. I have pushed a new commit with that change.